### PR TITLE
refactor algorithm 

### PR DESCRIPTION
**Pull request by ps3611**
refactoring structure of algorithm folder
works nicely

### CODE REVIEW

File: algorithm/index.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+  const rawData = await loadRawData();
   const avgM2Price = avgCalc(rawData);
   const formattedData = rawData.map((obj) => {
     const formattedObj = {};
     formattedObj.thumbnail = obj.thumbnail;
     formattedObj.price = obj.price;
     formattedObj.size = obj.size;
-    formattedObj.province = obj.province;
-    formattedObj.municipality = obj.municipality;
     formattedObj.country = obj.country;
```
Comment (alvaroha1) [inline algorithm/index.js:16]: This means we can filter by country?

Comment (ps3611) [reply algorithm/index.js]: no, red means i removed it

Comment (ps3611) [reply algorithm/index.js:16]: yes, country stays

File: algorithm/index.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+  diffCalc,
+} = require('./helpers');
 
 module.exports = async () => {
-  const rawData = await loadRawDataFromDb();
+  const rawData = await loadRawData();
   const avgM2Price = avgCalc(rawData);
   const formattedData = rawData.map((obj) => {
     const formattedObj = {};
     formattedObj.thumbnail = obj.thumbnail;
```
Comment (stevenspyramid) [inline algorithm/index.js]: please change the syntax to a destructuring syntax e.i. const formattedObj = {thumbnail, price, size, country};

**Review by stevenvana — CHANGES_REQUESTED** [review]
please change the syntax of formattedObj

**Review by stevenvana — APPROVED** [review]
great work!

**Pull request merged**